etc/kayobe/kolla/globals.yml

```diff
@@ -6,52 +6,56 @@
 # non-overcloud hosts
 enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts.os_family == 'Debian' }}{% endraw %}"
 
-magnum_tag: yoga-20230825T142202
-neutron_tag: yoga-20230728T081242
-nova_tag: yoga-20230718T112646
-octavia_tag: yoga-20230523T110936
-openvswitch_tag: yoga-20230515T150233
-ovn_tag: yoga-20230515T150233
-
 # kolla_base_distro must be set here to be resolvable on a per-host basis
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ 'centos' if ansible_facts.distribution == 'Rocky' and ansible_facts.distribution_major_version == '8' else ansible_facts.distribution | lower }}{% endraw %}"
 
 kayobe_image_tags:
   openstack:
-    centos: yoga-20230217T135826
+    centos: yoga-20230904T092025
     rocky: yoga-20230310T170929
     ubuntu: yoga-20230220T181235
   bifrost:
-    centos: yoga-20230217T160618
+    centos: yoga-20230904T142433
     rocky: yoga-20230310T194732
     ubuntu: yoga-20230220T184947
   blazar:
-    centos: yoga-20230315T125157
     rocky: yoga-20230315T130918
     ubuntu: yoga-20230315T125441
   caso:
-    centos: yoga-20230315T125157
     rocky: yoga-20230315T130918
     ubuntu: yoga-20230315T125441
   grafana:
-    centos: yoga-20230419T085955
     rocky: yoga-20230419T111514
     ubuntu: yoga-20230426T084340
   ironic:
-    centos: yoga-20230316T154655
     rocky: yoga-20230316T170311
     ubuntu: yoga-20230316T154704
   ironic_dnsmasq:
-    centos: yoga-20230217T135826
     rocky: yoga-20230310T170929
     ubuntu: yoga-20230220T181235
+  magnum:
+    rocky: yoga-20230825T142202
+    ubuntu: yoga-20230825T142202
+  neutron:
+    rocky: yoga-20230728T081242
+    ubuntu: yoga-20230728T081242
+  nova:
+    rocky: yoga-20230718T112646
+    ubuntu: yoga-20230718T112646
+  octavia:
+    rocky: yoga-20230523T110936
+    ubuntu: yoga-20230523T110936
   opensearch:
```

Comment:
The new opensearch image is going to be incompatible with kolla-ansible until we re-revert https://github.com/stackhpc/kolla-ansible/pull/427. But then Ubuntu/RL9 will be incompatible. So we'll need opensearch images for those distros too before we merge this :(

Reply:
Ahh, good catch. I think @bbezak was building new opensearch images, so was planning on rebasing on that.

Reply:
Done in: https://github.com/stackhpc/stackhpc-kayobe-config/pull/612/commits/ff6b46c8fb8a7d4e35cea1e8d9487cee257a41a8